lib/charms/ip_router_interface/v0/ip_router_interface.py

```diff
@@ -380,7 +382,7 @@ def get_flattened_routing_table(self) -> List[Network]:
 
     def _sync_routing_tables(self) -> None:
         """Syncs the internal routing table with all of the requirer's app databags"""
-        routing_table = self.get_flattened_routing_table()
```

Comment:
Do we expect get_flattened_routing_table to still be used? It's referenced in the docs above so I suspect that yes but please confirm

Reply:
I would prefer to remove it, but the spec shows an example output that's formatted this way, so I'm hesitant. How should I go about modifying the spec?

Reply:
The spec only cares about the content of the charm relation interface, not what is sent or received by the charm to the lib.

Reply:
So, do we still need this `get_flattened_routing_table` method or not? If yes, close this conversation, if no, please update the docs.